corehq/apps/geospatial/utils.py

```diff
@@ -33,6 +33,12 @@ def get_geo_user_property(domain):
     return config.user_location_property_name
 
 
+def get_celery_task_tracker(domain, base_key):
+    task_key = f'{base_key}_{domain}'
+    message_key = f'{base_key}_message_{domain}'
```

Comment:
Same as previous comment regarding separate task tracker key

Reply:
Addressed in [196ffd3](https://github.com/dimagi/commcare-hq/pull/35126/commits/196ffd3fd025e3a8a5999498ef91eae399c34225).